src/test/java/com/epam/tc/hw2/ExerciseTwoTest.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.tc.hw2;
+
+//import static com.epam.tc.hw2.ExerciseOneTest.*;
+import java.time.Duration;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+
+
+public class ExerciseTwoTest extends AbstractTest {
+    private static String ID_DROPDOWN_SERVICE_MENU = "dropdown";
+    private static String XPATH_MENU_ITEM_DIFFERENT_ELEMENT = "//a[text()='Different elements']";
+    private static String XPATH_CHECKBOX_WATER = "//label[text()[contains(., ' Water')]]/*[@type='checkbox']";
+    private static String XPATH_CHECKBOX_WIND = "//label[text()[contains(., ' Wind')]]/*[@type='checkbox']";
+    private static String XPATH_RADIOBUTTON_SILVER = "//label[text()[contains(., ' Silver')]]/*[@type='radio']";
+    private static String XPATH_DROPDOWN_BLUE = "//*[text()='Blue']";
+    private static String XPATH_LOGS = "//*[@class='panel-body-list logs']";
+
+
+    @Test
+    public void exercise() {
+        //1. Open test site by URL
+        driver.get(URL_HOME_PAGE);
+        //2. Assert Browser title "Home Page"
+        Assertions.assertThat(driver.getTitle())
+                .isEqualTo(HOME_PAGE);
+        //3. Perform login
+        WebElement loginDropdown = new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath(XPATH_LOGIN_DROPDOWN)));
+        loginDropdown.click();
+        WebElement loginInput = driver.findElement(By.xpath(XPATH_LOGIN_INPUT));
+        loginInput.sendKeys(loginText);
+        WebElement passwordInput = driver.findElement(By.xpath(XPATH_PASSWORD_INPUT));
+        passwordInput.sendKeys(passwordText);
+        WebElement loginSubmit = driver.findElement(By.xpath(XPATH_LOGIN_SUBMIT));
+        loginSubmit.click();
+        WebElement logoutButton = driver.findElement(By.xpath(XPATH_LOGOUT_SUBMIT));
+        boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
+        Assertions.assertThat(logoutButtonIsDisplayed)
```

Comment:
Why do you always use extra variables?
```
boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
Assertions.assertThat(logoutButtonIsDisplayed)
```
instead of
```
Assertions.assertThat(logoutButton.isDisplayed())
```
Is there any reason for it?